src/surface/flight_control/flight_control/manual_control_node.py

```diff
@@ -101,6 +104,16 @@ def __init__(self) -> None:
         self.seen_right_cam = False
         self.valve_manip_state = False
 
+        self.previous_pixhawk_instruction = PixhawkInstruction(
+            forward=float(0),
+            lateral=float(0),
```

Comment:
0.0 is better than float(0)

Reply:
Also these values get auto set to 0.0 anyway.

Reply:
Changed; should probably test